handlers/omni/v2/handler.go

```diff
@@ -75,15 +76,15 @@ func customParseFuncs(ctx *handlers.HandlerCtx) transform.CustomParseFuncs {
 }
 
 func fileFormats(ctx *handlers.HandlerCtx) []omniv2fileformat.FileFormat {
-	// If caller specifies a custom FileFormat, we'll use it (and it only);
-	// otherwise we'll use the builtin ones.
 	formats := []omniv2fileformat.FileFormat{
 		omniv2json.NewJSONFileFormat(ctx.Name),
 		omniv2xml.NewXMLFileFormat(ctx.Name),
-		// TODO more bulit-in omniv2 file formats to come.
+		// TODO more built-in omniv2 file formats to come.
 	}
-	if ctx.HandlerParams != nil && ctx.HandlerParams.(*HandlerParams).CustomFileFormat != nil {
-		formats = []omniv2fileformat.FileFormat{ctx.HandlerParams.(*HandlerParams).CustomFileFormat}
+	if ctx.HandlerParams != nil {
```

Comment:
Looks `ctx.HandlerParams` is just `interface{}`, do we need to consider the situation that it can't be converted to `HandlerParams` struct?

Reply:
right catch. fixed.